Header header

uint32 height
uint32 width

string dtype
uint8 n_channels

uint32 step
uint8[] data
